lib/transition-config/site.rb

```diff
@@ -153,6 +159,14 @@ def self.check_required_fields_present!(masks = MASKS)
       raise TransitionConfig::RequiredFieldsMissingException.new(missing) unless missing.empty?
     end
 
+    def self.check_homepage_protocol_present!(masks = MASKS)
+      missing = []
+      TransitionConfig::Site.all_with_basenames(masks).each do |site, _|
+        missing << site.abbr unless %w(http https).any? { |protocol| site.homepage.include?(protocol) }
```

Comment:
I just tested this by adding "http" at the _end_ of a homepage URL and it didn't complain, so `include?` might be better as `start_with?`?